projects/ui/src/lib/button/components/action-button-toggle/lab900-action-button-toggle.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, Input } from '@angular/core';
+import { ActionButton } from '../../models/action-button.model';
+import { ThemePalette } from '@angular/material/core';
+
+@Component({
+  selector: 'lab900-action-button-toggle',
+  templateUrl: './lab900-action-button-toggle.component.html',
+})
+export class Lab900ActionButtonToggleComponent {
+  @Input()
+  public action: ActionButton;
+
+  @Input()
+  public data: any;
+
+  public getSelected(): ActionButton {
+    for (const action of this.action.subActions) {
+      if ((typeof action.selected === 'function' && action.selected(this.data)) || action.selected === true) {
+        return action;
+      }
+    }
+    return null;
+  }
+
+  public getColor(): ThemePalette {
+    if (typeof this.action.color === 'function') {
+      return this.action.color(this.data);
+    }
+    return this.action.color;
```

Comment:
```suggestion
    return typeof this.action.color === 'function' ? this.action.color(this.data) : this.action.color
```
Misschien nog beter om ergens een util function te maken die dit checked want deze check hebben we nu wel al op veel plaatsen opnieuw.